.cirrus.yml

```diff
@@ -1,30 +1,31 @@
 FreeBSD_task:
-  freebsd_instance:
-    image: freebsd-12-1-release-amd64
+  matrix:
+    env:
+      SSL: openssl
+    env:
+      SSL: openssl111
+    env:
+      SSL: libressl
+    env:
+      SSL: libressl-devel
+    env:
+      SSL:
+  matrix:
+    freebsd_instance:
+      image: freebsd-12-1-release-amd64
+    freebsd_instance:
+      # 11-3-release doesn't boot: https://cirrus-ci.org/guide/FreeBSD/
+      image: freebsd-11-3-stable-amd64-v20191205
   env:
     ASSUME_ALWAYS_YES: TRUE # required for unattanded "pkg" invocation
-  install_script:
-    - pkg install cmake openssl111 git
+  prepare_script:
+    - pkg install cmake git $SSL
```

Comment:
can you, please, change it to 

`pkg install -y cmake openssl111 git`

and remove `ASSUME_ALWAYS_YES` variable

?

Reply:
done